Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,160 @@
+
+                Console.WriteLine(name + " is " + age + " years old, his weight is " + weight + " kg and his height is " + height + " cm.");
+
+                float BMI = CalculateBmi(weight, height / 100);
+
+                Console.WriteLine("Your BMI is " + BMI);
+
+                WeightType(BMI);
+
+                calculateAgain = ResponseValidation();
+            }
+
+            Console.WriteLine("Exiting...");
+
+        }
+
+        public static string PromptString(string message)
+        {
+            Console.Write(message);
+            var input = Console.ReadLine();
+
+            if (string.IsNullOrEmpty(input))
+            {
+                Console.WriteLine("Name cannot be empty.");
+                return "-";
+            }
+
+            return input;
+        }
+
+        public static int PromptInt(string message)
+        {
+            Console.Write(message);
+            var input = Console.ReadLine();
+
+            if (string.IsNullOrEmpty(input))
+            {
+                return 0;
+            }
+
+            if(int.TryParse(input, out int output) == false)
+            {
+                Console.WriteLine('"' + input + '"' + " is not a valid number.");
+                return -1;
+            }
+            return output;
+        }
+
+        public static float PromptFloat(string message)
+        {
+            Console.Write(message);
+            var input = Console.ReadLine();
+
+            if (string.IsNullOrEmpty(input))
+            {
+                return 0;
+            }
+
+            if (float.TryParse(input, out float output) == false)
+            {
+                Console.WriteLine('"' + input + '"' + " is not a valid number.");
+                return -1;
+            }
+
+            return output;
+        }
+
+        public static float CalculateBmi(float weight, float height)
+        {
+            if(weight <= 0 && height <= 0)
+            {
+                Console.WriteLine("Failed calculating BMI:");
+                Console.WriteLine("Weight cannot be equal or less than zero, but was " + weight + ".");
+                Console.WriteLine("Height cannot be equal or less than zero, but was " + height + ".");
+                return -1;
+            }
+            else if (weight <= 0)
+            {
+                Console.WriteLine("Failed calculating BMI:");
+                Console.WriteLine("Weight cannot be equal or less than zero, but was " + weight + ".");
+                return -1;
+            }
+            else if (height <= 0)
+            {
+                Console.WriteLine("Failed calculating BMI:");
+                Console.WriteLine("Height cannot be equal or less than zero, but was " + height + ".");
+                return -1;
+            }
+
+            return weight / (height * height);
+        }
+
+        public static void WeightType(float BMI)
+        {
+            if (BMI == -1)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Do not check floating point equality with exact values, use a range instead.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4885366)